### PR TITLE
Add score statistics

### PR DESCRIPTION
**Pull request by armstrongnate**
refs: MBl-14711
affects: student
release note: Added support for assignment score statistics

### CODE REVIEW

**Comment by armstrongnate** [comment]
@ndrewh, same deal here as iOS. Merging from this PR so that we can run CI checks. Your feedback is welcome.

File: apps/student/src/main/java/com/instructure/student/mobius/assignmentDetails/ui/gradeCell/GradeCellViewState.kt [hunk 1]
```diff
@@ -127,12 +137,44 @@ sealed class GradeCellViewState {
                 finalGrade = context.getString(R.string.finalGradeFormatted, submission.grade)
             }
 
+            val stats = assignment.scoreStatistics
+            if (stats != null) {
+                val minStr = "Low: ${NumberHelper.formatDecimal(stats.min, 1, true)}"
+                val maxStr = "High: ${NumberHelper.formatDecimal(stats.max, 1, true)}"
+                val meanStr = "Mean: ${NumberHelper.formatDecimal(stats.mean, 1, true)}"
```
Comment (armstrongnate) [inline apps/student/src/main/java/com/instructure/student/mobius/assignmentDetails/ui/gradeCell/GradeCellViewState.kt:144]: @Blackchai @JordanMarshall will these get localized? What's the correct way?

Comment (Blackchai) [reply apps/student/src/main/java/com/instructure/student/mobius/assignmentDetails/ui/gradeCell/GradeCellViewState.kt:144]: These needed to be added as strings in panda res, like this:
<string name="lowValue">Low: %d</string>

**Comment by armstrongnate** [comment]
Thanks for the refactors, @JordanMarshall. They look great to me. Since I'm the author I can't give the official approval. Thank you @Blackchai for taking a look.

**Comment by ndrewh** [comment]
This looks good to me as well.

**Comment by Blackchai** [comment]
I'll review the rest today.

**Review by Blackchai — APPROVED** [review]
LGTM!

**Comment by armstrongnate** [comment]
Thanks @Blackchai and thank you @ndrewh!

**Pull request merged**